client/src/main/scala/drt/client/components/FlightTableRow.scala

```diff
@@ -254,8 +254,8 @@ object FlightTableRow {
     <.span(^.className := "no-wrap", Tippy.interactive(<.span(^.key := "walk-time", walkTimeString), gateOrStand))
   }
 
-  def offScheduleClass(arrival: Arrival, considerPredictions: Boolean): String = {
-    val eta = arrival.bestArrivalTime(considerPredictions)
+  def offScheduleClass(arrival: Arrival): String = {
+    val eta = arrival.bestArrivalTime(considerPredictions = true)
```

Comment:
if considerPrediction will always be true , do we need it ?

Reply:
We don't need it, but it's defined drt-lib and I didn't want to scope creep too much